### ISSUE TITLE
Support Ionic Services

### ISSUE THREAD

**Issue by cfjedimaster** [driftyco/ionic-lab-issues#35]
This is _way_ early, but if I could see my stats, do pushes, and use deploy, from the app, that would be cooler than beer. Almost.

**Comment by jbavari** [comment]
Yes and yes. Side note: what is your favorite beer? Pretty sure I owe you one!

**Comment by cfjedimaster** [comment]
Next time I'm town. Just convince IBM we need to have special "high level meetings with exec types." I'll even bring a tie. (I won't put it on. But I'll bring the one tie I own.)

**Comment by NorthMcCormick** [comment]
Ah I second this. If the upload had a more extensive modal or it had a deploy button to choose the channel and note. The website works great for it right now though
